### PR TITLE
fix automation tests

### PR DESCRIPTION
**Pull request by anirudhwarrier**

### CODE REVIEW

File: integration-tests/benchmark/keeper_test.go [hunk 1]
```diff
@@ -314,7 +314,7 @@ func SetupAutomationBenchmarkEnv(t *testing.T, keeperTestConfig types.KeeperBenc
 		TTL: time.Hour * 720, // 30 days,
 		NamespacePrefix: fmt.Sprintf(
 			"automation-%s-%s-%s",
-			keeperTestConfig.GetConfigurationName(),
+			strings.ToLower(testConfig.ConfigurationName),
```
Comment (anirudhwarrier) [inline integration-tests/benchmark/keeper_test.go:317]:
```suggestion
			strings.ToLower(keeperTestConfig.GetConfigurationName()),
```

File: integration-tests/test.Dockerfile [hunk 2]
```diff
@@ -2,10 +2,9 @@ ARG BASE_IMAGE
 ARG IMAGE_VERSION=latest
 FROM ${BASE_IMAGE}:${IMAGE_VERSION}
 
-ARG SUITES=chaos migration performance reorg smoke soak benchmark
+ARG SUITES=chaos migration reorg smoke soak benchmark
```
Comment (anirudhwarrier) [inline integration-tests/test.Dockerfile:5]:
```suggestion
ARG SUITES=chaos migration performance reorg smoke soak benchmark
```

File: .github/workflows/automation-benchmark-tests.yml [hunk 3]
```diff
@@ -68,6 +73,8 @@ jobs:
           TEST_ARGS: -test.timeout 720h
           ENV_JOB_IMAGE: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ secrets.QA_AWS_REGION }}.amazonaws.com/chainlink-tests:${{ github.sha }}
           INTERNAL_DOCKER_REPO: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ secrets.QA_AWS_REGION }}.amazonaws.com
+          TEST_TYPE: ${{ github.event.inputs.testType }}
+          TEST_TEST_TYPE: ${{ github.event.inputs.testType }}
```
Comment (Tofel) [inline .github/workflows/automation-benchmark-tests.yml:77]: maybe better to add forwarding of `TEST_TYPE` variable to CTF?

**Review by Tofel — APPROVED** [review]
LGTM

**Pull request merged**